__tests__/integration/configuration/docker-compose.yml

```diff
@@ -67,6 +70,166 @@ services:
+  # Wallet Service API Lambda on 3000 and Wallet Service Websocket Lambda on 3001
+
+  mysql:
+    image: centos/mysql-80-centos7 # Necessary to use specific authentication protocol
+    networks:
+      - hathor-privnet
+    environment:
+      MYSQL_ROOT_PASSWORD: hathor
+      MYSQL_DATABASE: wallet_service
+      MYSQL_USER: wallet_service_user
+      MYSQL_PASSWORD: password
+      MYSQL_DEFAULT_AUTHENTICATION_PLUGIN: mysql_native_password
+    ports:
+      - "3380:3306"
+    healthcheck:
+        test: ["CMD", "mysql", "-h", "mysql", "-u", "wallet_service_user", "-ppassword", "-e", "SELECT 1", "wallet_service"]
+
+  redis:
+    image: redis:6.2
+    networks:
+      - hathor-privnet
+    ports:
+      - "6379:6379"
+    healthcheck:
+      test: [ "CMD", "redis-cli", "ping" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  ws-migrator:
+    image: hathornetwork/hathor-wallet-service-migrator
+    restart: "no"  # Critical: don't restart migration service
+    depends_on:
+      mysql:
+        condition: service_healthy
+    environment:
+      DB_ENDPOINT: "mysql"
+      DB_NAME: "wallet_service"
+      DB_USER: "wallet_service_user"
+      DB_PASS: "password"
+      DB_PORT: 3306
+    networks:
+      - hathor-privnet
+
+  ws-daemon:
+    image: hathornetwork/hathor-wallet-service-sync-daemon
+    depends_on:
+      ws-migrator:
+        condition: service_completed_successfully
+      fullnode:
+        condition: service_healthy
+      mysql:
+        condition: service_healthy
+    environment:
+      # Those two variables are used to set all the necessary dynamic env variables for this pristine environment
+      FETCH_FULLNODE_IDS: true
+
+      # Connection to other services in this private network
+      FULLNODE_HOST: "fullnode:8080"
+      FULLNODE_NETWORK: "nano-testnet-bravo"
+      REDIS_URL: redis://redis:6379
+      DB_ENDPOINT: "mysql"
+      DB_NAME: "wallet_service"
+      DB_USER: "wallet_service_user"
+      DB_PASS: "password"
+      DB_PORT: 3306
+
+      # Other env vars. Some of them may not be necessary and could be removed in the future
+      USE_SSL: "false"
+      NEW_TX_SQS: ""
+      PUSH_NOTIFICATION_ENABLED: "false"
+      WALLET_SERVICE_LAMBDA_ENDPOINT: "" # XXX: Confirm if this is really not used
+      ACCOUNT_ID: 1234
+      ALERT_MANAGER_TOPIC: "alert-topic"
+      ALERT_MANAGER_REGION: "us-east-1"
+      APPLICATION_NAME: "hathor-wallet-service"
+      NODE_ENV: "test-privnet"
+      STAGE: "local"
+      MAX_ADDRESS_GAP: 10
+      NETWORK: "privatenet"
+#      BLOCK_REWARD_LOCK: 1
+    ports:
+      - "8081:8081"
+      - "8082:8082"
+    networks:
+      - hathor-privnet
+
+  ws-serverless:
+    image: hathornetwork/hathor-wallet-service-service
+    depends_on:
+      fullnode:
+        condition: service_healthy
+      mysql:
+        condition: service_healthy
+      ws-daemon:
+        condition: service_started # TODO: Implement healthcheck
```

Comment:
As soon as we have an initial version published, those important improvements can be worked upon